aruna/api/hooks/services/v1/hooks_service.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package aruna.api.hooks.services.v1;
+option go_package = "github.com/ArunaStorage/go-api/aruna/api/storage/services/v1";
+option java_multiple_files = true;
+option java_package = "com.github.ArunaStorage.java_api.aruna.api.storage.services.v1";
+option java_outer_classname = "HooksService";
+
+import "aruna/api/storage/models/v1/models.proto";
+
+// HooksService
+//
+// Status: ALPHA
+//
+// A service that enables automatic Hook scheduling
+service HooksService {
+    rpc CreateHook(CreateHookRequest) returns (CreateHookResponse) {}
+    rpc ListHook(ListHookRequest) returns (ListHookResponse) {}
```

Comment:
After taking a look at the ` ListHookRequest` definition I assume this lists all Hooks which are available in a specific project. So I would rather use the plural for the names.

```suggestion
    rpc ListHooks(ListHooksRequest) returns (ListHooksResponse) {}
```